### DISCUSSION TITLE
Optional Thinking Cap

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#17] [Extra Credit]
# Extra Thinking Cap

### Due 5/9

>    ## How can local communities and take ownership of technology and surveys related to themselves?

Reflect on how we are starting to think about collecting data, think about what it means to empower local communities and make the tools their own.

#### Note
Because this is an optional thinking cap, commenting on other's post is not required, but those that do will get extra credit for doing so.

**Comment by cassiuscarandang** [comment]
Local communities are flourishing everyday through technology. Social media is our new promotional tool and daily news, education is normalizing computers and laptops as tools for academic success, and technology/surveys are being used to enhance community engagement. But with the technological advancements occurring everyday, mishaps and abuse of power can cause invalidness and misconceptions within how data and information can be presented. In order for local communities to take ownership of themselves, the involvement of the community must happen. This comes from engaging in developing surveys that can only be validated by the community if they are ones that are going to represent the data. Watching the lecture today (5/2) about our guest speakers from CAUSE, really caught my attention when they mentioned a ton about political and civic empowerment through Asian American communities. How a lot of their work comes from receiving information from neighborhoods themselves and utilizing the collected data towards creating maps that result in effective impact in self-empowerment. In order to formulate social change, it can be done through the community and how you represent them. Especially throughout the technological age we live in today.

**Reply by kmtsujimoto** [reply]
I also really appreciated the importance of political and civic engagement introduced by CAUSE in the guest lecture. I was especially intrigued by their emphasis on voting power in the projects they created and advocated for in the drawing of California's voting districts, which helped me realize the importance of GIS to relevant social and political issues within my own community. The active role that CAUSE took through activism was not only a voice for Asian Americans itself but also helped elect an Asian American official who could give Asian Americans even more of a voice on a national stage. In this way, taking ownership of technology and surveys allowed CAUSE to empower themselves and Asian American communities.

**Comment by kmtsujimoto** [comment]
I believe that the most important element of local communities having and taking ownership of technology and surveys related to themselves is access. There is still a lot of room to improve to provide truly equitable technology and surveys for communities and individual community members.

One major barrier that is especially applicable to surveys is language-based access. Especially for many AAPI communities and recent immigrant communities of color, language access can affect community members' ability to understand and take surveys, limiting their voice and political power. Survey makers within communities should include easily available language options in order to improve accessibility to their surveys. In turn, community members and allies should advocate for better translations and further language considerations in both externally and internally created community surveys.

Another barrier is access to the technology that is required to make and take digital surveys. Especially as the world continues to move into a more and more automated and digitalized world, we behind to lose some of the benefits of survey methods that are being considered obsolete. Paper surveys and ballots for example are becoming less common, but to stop offering these options would potentially limit those who cannot afford or struggle to navigate computers and other digital tools. In some cases, underserved and underprivileged communities who lack access are the populations that need these community surveys the most. Survey makers and survey takers need to consider the biases that may come from access limitations in surveys on the basis of technology.

Like consumers, members of communities have power over survey makers. They can decide which surveys they will support. They can advocate against surveys that don't do justice to their communities. They can lobby government officials who influence and create surveys, and they can create surveys themselves to emphasize the issues that are truly important to the community.

**Comment by danigold1020** [comment]
I believe that for local communities to take ownership and empower themselves, they should actively try to stay updated with advancements in technology and help each other learn new technologies. It is important for them to try preventing the gap between them and the elite/academic institutions that posses the knowledge of such technologies. This may be difficult since new software is often expensive/requires a subscription, is hard to learn, or takes time to learn/master. Thus, if they wish to take ownership, it is important to  voice their desire to learn new technologies to obtain access to them (which is still not guaranteed), as well as conduct surveys themselves. Since surveys are powerful tools that must be carefully designed to obtain unbiased (or less biased) results, it is important for those who understand the population of survey respondents most to be the ones creating the questions so that the survey will be more accurate and the respondents will feel more comfortable answering questions that are being asked from others in their own community, enabling them to be more open. Surveys are a method of data collection, which many local communities may be hesitant to participate in, thus emphasizing the importance for these communities to design and distribute their own surveys within their communities to build trust and strengthen the communities' sense of empowerment.

**Comment by mandymwlai** [comment]
I think the first step for local communities to take ownership of technology is to ensure that the data is available to the community. Too often data in a survey is harvested for academic purposes and they are never to be used again after a particular paper is published. Once the data is returned to the community, it can be used for social goods such as local policy-making or building of infrastructure. For example, the data could be used to identify issues in the community that need to be addressed, such as lack of access to healthcare services during COVID-19. Policy-makers can then make informed decisions and develop programs to address these health-related issues. Finally, local communities should be kept informed in the data-collection process to ensure that voices of the community are heard. It would be a good idea to include a community member in the research team that provides their insights. The participant can voice their concerns regarding the project or if their community is being misrepresented.

**Reply by kmtsujimoto** [reply]
I agree that it is really important for data to be made available to community members so they can use it properly to best serve their communities. However, I think it may be difficult in practice to implement this strategy. This connects to our discussion on open-source and closed-source developing. One of the major issues was that closed-source developing, which may include closed-source data gathering and analysis, is more profitable for large corporations that aim to capitalize on community data. In order to accomplish more of an open-sourced data system, we may need political activism and advocacy to develop standards of ethics that large corporations are forced to follow to the benefit of the community. It may, however, also be problematic for data to be too publicly available. It's possible for outside groups to abuse community data in ways that do not properly serve the community. In that way, data is a double-edged sword.

**Reply by danigold1020** [reply]
I really like that you said that community members should be involved in the research team and I would make sure that they are included in the survey design/data collection process as well to build trust and empowerment in the community. The data should definitely be returned to the communities so that they can feel involved in the learning process rather than a source of data extraction, never to see the fruits of the information they provide to a larger movement/research study.
